orchestration/iam.go

```diff
@@ -43,7 +42,29 @@ func (o *Orchestrator) DefaultTaskExecutionPolicy(path string) im.PolicyDoc {
 				Resource: []string{
 					fmt.Sprintf("arn:aws:secretsmanager:*:*:secret:spinup/%s/*", path),
 					fmt.Sprintf("arn:aws:ssm:*:*:parameter/%s/*", path),
-					fmt.Sprintf("arn:aws:kms:*:*:key/%s", o.IAM.DefaultKmsKeyID),
+					fmt.Sprintf("arn:aws:kms:*:*:key/%s", kms),
+				},
+			},
+			{
+				Effect: "Allow",
+				Action: []string{
+					"elasticfilesystem:ClientRootAccess",
+					"elasticfilesystem:ClientWrite",
+					"elasticfilesystem:ClientMount",
+				},
+				Resource: []string{"*"},
+				Condition: yiam.Condition{
+					"Bool": yiam.ConditionStatement{
+						"elasticfilesystem:AccessedViaMountTarget": []string{"true"},
+					},
+					"StringEqualsIgnoreCase": yiam.ConditionStatement{
+						"aws:ResourceTag/spinup:org": []string{
+							"${aws:PrincipalTag/spinup:org}",
```

Comment:
👀 is this right?

Reply:
❓ I'm not sure what you mean, which part?  i think it's right.... lol.

Reply:
this says the value of the `spinup:org` tag is the same on the resource (efs) and the principal (role in this case)

Reply:
I was wondering about the `$` is it some kind of interpolation?

Reply:
yeah exactly, it's kinda unusual 😆 

https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction_attribute-based-access-control.html

Reply:
ah, cool, it's the ABAC stuff